tcadmin/diff.py

```diff
@@ -17,6 +18,28 @@
 t = blessings.Terminal()
 
 
+def fast_diff(left, right, n):
+    left_file = tempfile.NamedTemporaryFile("w")
+    left_file.write("\n".join(left))
+    right_file = tempfile.NamedTemporaryFile("w")
+    right_file.write("\n".join(right))
+    output = subprocess.run(
+        [
+            "diff",
+            f"-U{n}",
+            "--label",
+            "current",
+            "--label",
+            "generated",
+            left_file.name,
+            right_file.name,
+        ],
+        encoding="utf8",
+        stdout=subprocess.PIPE,
+    ).stdout
```

Comment:
We may want to clean up the temp files here?

Reply:
I'm writing a follow-up PR to clean up tempfiles